packages/lodestar/src/db/api/beacon/repositories/stateArchive.ts

```diff
@@ -7,16 +6,12 @@ import {IDatabaseController} from "../../../controller";
 import {Bucket} from "../../schema";
 import {Repository} from "./abstract";
 
-export class StateArchiveRepository extends Repository<Epoch, TreeBacked<BeaconState>> {
+export class StateArchiveRepository extends Repository<Epoch, BeaconState> {
```

Comment:
we want to store BeaconState data in the end. If consumer wants TreeBacked, it can build TreeBacked manually like in `chain.waitForState()`